supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -280,11 +280,11 @@
 [discrete]
 [[grub]]
 ==== image:images/yes.png[yes] GRUB (noun)
-*Description*: "GRUB" is an acronym for "GRand Unified Bootloader", which is a Linux boot loader.
+*Description*: "GRUB" is an acronym for "GRand Unified Bootloader", which is a Linux boot loader. (GRUB 2 has replaced what was formerly known as GRUB, i.e. version 0.9x, which has, in turn, become GRUB Legacy.)
```

Comment:
I'd recommend getting rid of the parentheses around the sentence too and removing the "i.e.". The sentence is a bit complex, so my suggestion breaks it up into two sentences, but there are other solutions that could work I'm sure!

```suggestion
*Description*: "GRUB" is an acronym for "GRand Unified Bootloader", which is a Linux boot loader. Note that GRUB 2 has replaced what was formerly known as GRUB, which was version 0.9x. GRUB version 0.9x is now known as _GRUB Legacy_.
```

Reply:
Great, I like your version better! Could you please push it to the repo?

Reply:
@jheves I'd recommend that you make the change manually from your terminal/editor - just in case there are any further updates that need to be made, then your local branch and this PR aren't out of sync

Reply:
@bergerhoffer OIC! Consider it done :)